google/async_resumable_media/requests/_request_helpers.py

```diff
@@ -126,10 +132,9 @@ async def http_request(
         ~requests.Response: The return value of ``transport.request()``.
     """
 
-    #TODO(anirudhbaddepu) - look at default connect timeout and default read timeout
+    # TODO(anirudhbaddepu) - look at default connect timeout and default read timeout
```

Comment:
can you expand this comment?

Reply:
Added to the comment - the sync version uses a tuple for the defualt connect
    timeout and read timeout, since async requests can only take a single value and
    not a tuple we take the single connect timeout, but the logic technically diverges
    from the sync implementation here.